tensorboard/plugins/mesh/tf_mesh_dashboard/mesh-loader.js

```diff
@@ -153,6 +153,11 @@ Polymer({
         this.appendChild(this._meshViewer.getRenderer().domElement);
         this._meshViewerAttached = true;
       }
+    }).catch(function (error) {
+      if (!error || !error.code || error.code != vz_mesh.ErrorCodes.CANCELLED) {
+        error = error || 'Response processing failed.';
```

Comment:
Please only throw `Error`s, not string literals. Without an `Error`, you
lose programmatic access to the stack trace, and surrounding code that
attempts to reference `e.name` and `e.message` will fail. See:
<https://google.github.io/styleguide/jsguide.html#features-exceptions>